docs/source/backend.rst

```diff
@@ -3,44 +3,88 @@
+
+   The following table summarizes the timeline for the changes and deprecations.
+
+   +--------------------+--------------------------+--------------------------+--------------------------+
+   | **Backend**        | **0.7.0**                | **0.8.0**                | **0.9.0**                |
+   +====================+==========================+==========================+==========================+
+   | ``"sox"``          | Default if C++ extension | Available                | Removed                  |
+   | (deprecated)       | is available             |                          |                          |
+   +--------------------+--------------------------+--------------------------+--------------------------+
+   | ``"sox_io"``       | Available                | Default if C++ extension | Default if C++ extension |
+   |                    |                          | is available             | is available             |
+   +--------------------+--------------------------+--------------------------+--------------------------+
+   | ``"soundfile"``    | Default if C++ extension | Available                | Removed                  |
+   | (legacy interface, | is not available         |                          |                          |
+   | deprecated)        |                          |                          |                          |
+   +--------------------+--------------------------+--------------------------+--------------------------+
+   | ``"soundfile"``    | Available                | Default if C++ extension | Default if C++ extension |
+   | (new interface)    |                          | is not available         | is not available         |
+   +--------------------+--------------------------+--------------------------+--------------------------+
 
 Common Data Structure
 ~~~~~~~~~~~~~~~~~~~~~
 
-Structures used to exchange data between Python interface and ``libsox``. They are used by :ref:`sox<sox_backend>` and :ref:`soundfile<soundfile_backend>` but not by :ref:`sox_io<sox_io_backend>`.
+Structures used to report the metadata of audio files.
+
+AudioMetaData
+-------------
+
+.. autoclass:: torchaudio.backend.common.AudioMetaData
+
+SignalInfo (Deprecated)
+-----------------------
 
 .. autoclass:: torchaudio.backend.common.SignalInfo
 
-.. autoclass:: torchaudio.backend.common.EncodingInfo               
+EncodingInfo (Deprecated)
+-------------------------
+
+.. autoclass:: torchaudio.backend.common.EncodingInfo
 
 .. _sox_backend:
 
-Sox Backend
-~~~~~~~~~~~
+Sox Backend (Deprecated)
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-``sox`` backend is available on ``torchaudio`` installation with C++ extension. It is currently not available on Windows system.
+``"sox"`` backend is available on ``torchaudio`` installation with C++ extension. It is currently not available on Windows system.
```

Comment:
The sox backend... installations with the C++ extension. It is not available on Windows.